Monika After Story/game/event-handler.rpy

```diff
@@ -301,7 +301,8 @@ init 4 python:
         "STY": store.mas_stories.story_database,
         "CMP": store.mas_compliments.compliment_database,
         "FLR": store.mas_filereacts.filereact_db,
-        "APL": store.mas_apology.apology_db
+        "APL": store.mas_apology.apology_db,
+        "WRS": store.mas_windowreacts.windowreact_db
```

Comment:
can you add a TODO in zz_dm to add this database to that? dont actually add it because we dont have any new dms  yet